Formula/macvim.rb

```diff
@@ -100,16 +100,15 @@ def install
   end
 
   def caveats
-    if build.with?("python") && build.with?("python3")
+    if build.with?("python") && build.with?("python@2")
       <<~EOS
         MacVim can no longer be brewed with dynamic support for both Python versions.
-        Only Python 3 support has been provided.
+        Only Python 2 support has been provided.
       EOS
     end
   end
 
   test do
-    ENV.prepend_path "PATH", Formula["python"].opt_libexec/"bin"
     # Simple test to check if MacVim was linked to Python version in $PATH
     if build.with? "python"
       system_framework_path = `python-config --exec-prefix`.chomp
```

Comment:
Minor thing but `system_framework_path` no longer makes sense as a reference given the system doesn't ship any kind of `python3`.